### DISCUSSION TITLE
How to make a rounded Table

### DISCUSSION THREAD

**Discussion by remidiy** [chakra-ui/chakra-ui#3613] [Q&A] [answered]
The Table component doesn't seem to use borderRadius or rounded prop to make it  rounded. 

I tried `extendTheme` too but still not working.

Can anyone help me in achieving this ?

**Comment by malashevskyi** [comment, marked as answer]
It is hard to understand what you are trying to do without an example.

**Reply by remidiy** [reply]
`<Table rounded="md">{....}<Table>` this doesn't make the component corners rounded. Is there any other way to make them rounded ?

**Reply by nikolovlazar** [reply]
Like this: https://codesandbox.io/s/nameless-hooks-f8mxz?file=/src/index.tsx ?

**Reply by remidiy** [reply]
Thanks a lot for the help. It works like a charm.

**Reply by nikolovlazar** [reply]
You got it @remidi!

**Reply by befitsandpiper** [reply]
This codesandbox gets a typeError now unless you use the newest Chakra version.

**Reply by zulnabil** [reply]
you can add `overflow="hidden"` to the Box wrapper to make it work even when the table has colors

> Like this: https://codesandbox.io/s/nameless-hooks-f8mxz?file=/src/index.tsx ?

**Comment by malashevskyi** [comment]
You can use a worked example, that **lazarnikolov94** gave, but if you don't want to use Box wrapper, you need to set ```d="inline-block"``` for table and then you can use `borderRadius="md"` or `rounded="md"` [example](https://codesandbox.io/s/chakra-ui-table-border-radius-lpc4d?file=/src/App.js)

**Reply by remidiy** [reply]
Thanks for the help @malashevskyi , the solution you provided works but I need to flex the table. So I went with @lazarnikolov94
